nightfall-optimist/src/services/challenges.mjs

```diff
@@ -108,7 +133,7 @@ export async function createChallenge(block, transactions, err) {
     }
     // Challenge wrong root
     case 1: {
-      logger.debug('Challenging incorrect root');
+      logger.debug(`Challenging incorrect leaf count for block ${JSON.stringify(block, null, 2)}`);
```

Comment:
This logger should be `Challenging incorrect root`

Reply:
changed